api/src/main/java/io/strimzi/api/kafka/model/balancing/BrokerCapacity.java

```diff
@@ -42,6 +43,8 @@
     private String outboundNetwork;
     private Map<String, Object> additionalProperties = new HashMap<>(0);
 
+    @Deprecated
+    @DeprecatedProperty(description = "The Cruise Control disk capacity setting has been removed")
```

Comment:
AFAIU they are deprecated and ignored but not removed so the user can still set them but the operator won't use them. Is that right? Should we clarify in this description?